snowdrop-block/src/Snowdrop/Block/Fork.hs

```diff
@@ -90,28 +85,30 @@ verifyFork BlkStateConfiguration{..} osParams fork@(OldestFirst altHeaders) = do
       else RejectFork
   where
     loadBlocksFromTo
-        :: Int -> Maybe blockRef -> Maybe blockRef
-        -> m (NewestFirst [] (Blund header rawPayload undo))
+        :: Int -> Maybe (BlockRef blkType) -> Maybe (BlockRef blkType)
+        -> m (NewestFirst [] (Blund (Header blkType) (RawPayload blkType) (BlkUndo blkType)))
```

Comment:
it's better to create type alias `RawBlund blkType = (Blund (Header blkType) (RawPayload blkType) (BlkUndo blkType)` in Types.hs and use it